### ISSUE TITLE
Screen reader does not convey any confirmation message after activating 'Undo and Redo button' present in bottom.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#34]
User Experience:
If user is not notified by AT, then it will be hard to understand for visually impaired user that what UI changes happened on the page.

Re-pro steps:
1. Open Azure portal with valid credentials and create a API Management.
2. Select Developer Portal (Preview) button and select it.
3. Navigate to the https://accessibility.developer.azure-api.net/reference
4. Enable screen reader navigate to the Bottom nav and Select many option 'Save changes/ Undo/ Redo'.
5. Verify the issue.

Actual Result:
Screen reader does not convey any confirmation message after activating 'Undo and Redo button' present in bottom.

Expected Result:
Screen reader should provide a confirmation message about the update, screen reader should announce as ' Redo and Undo successfully'.

Note:
1. Similar issue exist with NVDA

MAS Reference:
MAS 1.3.1 - Info and Relationships (40)
